test/Renci.SshNet.Tests/Classes/BaseClientTest_Connected_KeepAliveInterval_NotNegativeOne.cs

```diff
@@ -56,12 +56,12 @@ protected override void Act()
         {
             _client.KeepAliveInterval = _keepAliveInterval;
 
-            // allow keep-alive to be sent a few times. .NET 7 is faster and
+            // allow keep-alive to be sent a few times. .NET 8 is faster and
             // we need to wait less because we want exactly three messages in a session.
 #if NETFRAMEWORK
             Thread.Sleep(195);
 #else
-            Thread.Sleep(180);
+            Thread.Sleep(170);
```

Comment:
We should really get rid of timing in tests...

Reply:
Agree. Better to create another PR...